docs/getting-started/config-env.md

```diff
@@ -1,23 +1,32 @@
 ---
-title: Configure Environment Variables
+title: Configuring Environment Variables
 ---
 
-# Configure environment variables/secrets
+# Configuring environment variables and secret keys
 
-Take a look at `Envfile`. This file lists all the `ENV` variables we use and provides a fake default for any missing keys. You'll need to get your own free [Algolia credentials](/backend/algolia) to get your development environment running.
+Take a look at `Envfile`. This file lists all the `ENV` variables we use and provides a fake default for any missing keys. You'll need to get your own free [Agolia credentials][agolia] to get your development environment running.
```

Comment:
`Agolia` and `agolia` are missing an `L`. The name io the company is [Algolia](https://www.algolia.com/) :)

Reply:
lol